sebs/cli.py

```
@@ -455,32 +455,40 @@ def package(
+):
     """Run regression test suite across benchmarks."""
+
     # for regression, deployment client is initialized locally
     # disable default initialization
+
+    from pathlib import Path
+
+    if Path(kwargs["cache"]) == Path("cache"):
+        kwargs["cache"] = os.path.join(os.path.curdir, "regression-cache")
+
     (config, output_dir, logging_filename, sebs_client, _) = parse_common_params(
```

Ruff (0.15.6)

[warning] Unpacked variable `output_dir` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)